bootstrap/sponge/pom.xml

```diff
@@ -77,10 +77,6 @@
                                     <pattern>com.google.guava</pattern>
                                     <shadedPattern>org.geysermc.geyser.platform.sponge.shaded.google.guava</shadedPattern>
                                 </relocation>
-                                <relocation>
```

Comment:
Make sure Kyori isn't still included in the jar - see how we exclude it in Velocity.